charts/kyverno/charts/crds/templates/crds.yaml

```diff
@@ -1,12 +1,12 @@
-{{- if .Values.crds.install }}
+{{- if .Values.install }}
```

Comment:
We can remove this part as the decision of installing it or not depends now on `conditions: crds.install` in chart.yaml of the parent one.

Reply:
It's generated by the make command automatically, you may need to update the make target if desired.

Reply:
Oh got it.

Reply:
I have removed it from the Make target.